charts/promitor-agent-scraper/README.md

```diff
@@ -1,17 +1,21 @@
 # Promitor
 
-[Promitor](https://promitor.io/) is an Azure Monitor scraper for Prometheus providing a scraping endpoint for Prometheus that provides a configured subset of Azure Monitor metrics.
+[Promitor](https://promitor.io/) is an Azure Monitor scraper for Prometheus providing
+a scraping endpoint for Prometheus that provides a configured subset of Azure
+Monitor metrics.
 
 ## TL;DR
 
 ```console
-$ helm repo add promitor https://promitor.azurecr.io/helm/v1/repo
```

Comment:
Is this because it's blocked or a suggestion?

Reply:
It's because the command has no output. The default rules have an error if you put a dollar in front of commands with no output.

For example, the following is ok:

```console
helm repo add promitor https://promitor.azurecr.io/helm/v1/repo
```

So is the following:

```console
$ helm repo add promitor https://promitor.azurecr.io/helm/v1/repo
"promitor" has been added to your repositories
```

But the following causes an error:

```shell
$ helm repo add promitor https://promitor.azurecr.io/helm/v1/repo
```

I actually think it makes sense, but I'm happy to disable that rule if you want. I just figured I'd start with the defaults.

Reply:
No worries, I was just curious - Thanks for explaining!